Update README.md adding the example of usage of -p,--parameter <param-name param-value> (#54)
mulesoft-labs/data-weave-cli: README.md

}
```

> ### Using parameters with the map `params`

Reviewer: ```suggestion
### Using Parameters
```

> Using the internal map `params`, we can access to an injected parameter in the command line with the -p option

Reviewer: ```suggestion
Using the internal map `params`, we can access injected parameters in the command line with the `-p` option:
```

```

> dw -p myName Julian "output applcation/json --- { name : params.myName }"

Reviewer: ```suggestion
dw -p myName Julian "output json --- { name : params.myName }"
```
